docs/src/pages/components/material-icons/SearchIcons.js

```diff
@@ -478,7 +491,25 @@ export default function SearchIcons() {
           />
         </Paper>
         <Typography className={classes.results}>{`${icons.length} matching results`}</Typography>
-        <Icons icons={icons} classes={classes} handleClickOpen={handleClickOpen} />
+        <FormControl className={classes.rotate}>
```

Comment:
I'm not sure about the option to toggle the rotation in the search page. A standalone demo could be enough, one with an icon where it make sense.

Reply:
Ok, I wasn't sure where we have the examples, but that makes sense, will move it there and will add it for the font icon too.